app/standalone.go

```diff
@@ -113,7 +113,7 @@ func doStandaloneInstallStatesDownload(art io.ReadCloser, key []byte,
 		// No doStandaloneFailureStates here, since we have not done anything yet.
 		return nil, err
 	}
-	installer, installers, err := installer.ReadHeaders(art, dt, key,
+	installer, installers, err := installer.ReadHeaders(art, dt, device.Config.GetVerificationKeys(),
```

Comment:
Linter is complaining that this line is too long; just put the last argument on a new line.